cache/redis/redis.go

```diff
@@ -71,24 +73,29 @@ func New(config map[string]interface{}) (rcache cache.Interface, err error) {
 		return nil, fmt.Errorf("redis did not respond with 'PONG', '%s'", pong)
 	}
 
+	// the config map's underlying value is int
```

Comment:
If we change this to look for a uint we can drop the less than 0 check on line 82